### DISCUSSION TITLE
第三方平台收藏歌单导入及cookie设置

### DISCUSSION THREAD

**Discussion by qinfengge** [GeekLee2012/Less-Player-Desktop#19] [Ideas]
1. 类似LX的导入设置，可以把自己的**喜欢**导入
![image](https://github.com/user-attachments/assets/37e374a7-681a-44e6-9b6d-90f2ed9ee509)

2. VIP歌曲无法播放，也许需要一个第三方平台的登录功能？
3. 一个歌词的小问题，桌面歌词置顶始终无效，歌词默认为非透明，仅锁定时才透明

**Comment by GeekLee2012** [comment]
你好，感谢反馈！
1. 歌单导入：类似LX的操作方式，现在不支持，待后续再想想实现方式；目前解决方式：如果是平台歌单（非自建），可通过歌单名搜索，然后收藏，操作步骤会繁琐些；
2. VIP歌曲，登录等功能：版权敏感度太高，暂时不会去实现；
3. 桌面歌词：置顶无效问题待排查（可能系统兼容问题）；歌词透明度，后续会跟进完善。
